src/core/tokenizerConfiguration.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * TODO: This is just an ugly mock since that tokenizer doesn't have access to the graph/ide
+ * Remove this once https://github.com/cursorless-dev/cursorless/issues/785 is implemented
+ */
+
+import * as vscode from "vscode";
+
+const defaultGetWordSeparators = (languageId: string) => {
+  return vscode.workspace
+    .getConfiguration("cursorless", { languageId })
```

Comment:
Should also add a comment to https://github.com/cursorless-dev/cursorless/blob/c914d477c9624c498a47c964088b34e484eac494/src/core/Decorations.ts#L58 indicating that we can't use a fine-grained settings listener until we move to graph